tests/integration/references/belongs-to-test.js

```diff
@@ -230,7 +230,7 @@ test("push(promise)", function(assert) {
     push = familyReference.push(deferred.promise);
   });
 
-  assert.ok(push.then, 'BelongsToReference.push returns a promise');
+  assert.ok(push instanceof Ember.RSVP.Promise, 'BelongsToReference.push() returns a promise');
```

Comment:
maybe `! instanceof PromiseProxyObject` ? might be the most clear, as well as updating the description.